src/benchmarker.js

```diff
@@ -172,7 +186,7 @@ function dump(
     beforeStats
   );
 
-  if (code.length <= 1000 && !outputFilename) {
+  if (code.length <= 1000 && typeof outputFilename === "undefined") {
```

Comment:
outputFilename !== undefined && outputFilename !== ""

Reply:
Is this not the case when an output filename is not provided? Wouldn't it be `outputFilename === undefined || outputFilename === ""`?

Reply:
You are correct.